src/frontend/citizen-portal/src/app/shared/components/resolution-header/resolution-header.component.ts

```diff
@@ -28,8 +26,8 @@ export class ResolutionHeaderComponent implements OnInit {
     this.params = this.route.snapshot.queryParams as QueryParamsForSearch;
     if (this.ticket) {
       this.ticketNumber = this.ticket.ticket_number;
-      this.time = this.datePipe.transform(this.ticket.issued_date, "HH:mm");
-      this.date = this.datePipe.transform(this.ticket.issued_date, "yyyy-MM-dd");
+      this.time =this.ticket.issued_date.substring(11,16); // hh:mm
```

Comment:
this.datePipe.transform(input.issued_date, "HH:mm", "UTC")
this.datePipe.transform(input.issued_date, "yyyy-MM-dd", "UTC")